prober/http.go

```diff
@@ -221,6 +224,14 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 		return false
 	}
 
+	jar, err := cookiejar.New(&cookiejar.Options{PublicSuffixList: publicsuffix.List})
+	if err != nil {
+		level.Error(logger).Log("msg", "Error generating cookiejar", "err", err)
+		return false
+	}
+	client.Jar = jar
+	client.Timeout = module.Timeout
```

Comment:
Where is this change coming from?

Reply:
Whoops, I mistakenly added that line about timeouts!